quickwit/quickwit-jaeger/src/lib.rs

```diff
@@ -66,17 +66,27 @@ const TRACE_INDEX_ID: &str = "otel-trace-v0";
 /// A base64-encoded 16-byte array.
 type TraceId = String;
 
+type TimeIntervalSecs = RangeInclusive<i64>;
+
 type JaegerResult<T> = Result<T, Status>;
 
 type SpanStream = ReceiverStream<Result<SpansResponseChunk, Status>>;
 
 pub struct JaegerService {
     search_service: Arc<dyn SearchService>,
+    lookback_period: Duration,
+    max_trace_duration: Duration,
+    max_retrieved_spans: u64,
 }
 
 impl JaegerService {
     pub fn new(search_service: Arc<dyn SearchService>) -> Self {
-        Self { search_service }
+        Self {
+            search_service,
+            lookback_period: Duration::from_secs(3 * 24 * 3600),
```

Comment:
I'm about to make this configurable in my next PR.

Reply:
I'm not sure to understand how this is used in the query.